broker/service/api/__init__.py

```diff
@@ -99,3 +108,30 @@
 except Exception as e:
     print "Error: %s" % e.message
     quit()
+
+""" Gets the IP address of one a the node contained
+    in a Kubernetes cluster
+
+    Raises:
+        Exception -- It was not possible to connect with the
+        Kubernetes cluster.
+
+    Returns:
+        string -- The node IP
+    """
+
```

Comment:
unnecessary blank line.

Reply:
also make it clear somewhere that you're getting always the last node listed in order to avoid the master nodes.

Reply:
Modification made in commit 540b521953feea4e73758f6d8173a1c58e9e65be